ui/src/main.rs

```diff
@@ -417,11 +426,35 @@ fn app(cx: Scope) -> Element {
             style { "{UIKIT_STYLES} {APP_STYLE} {theme}" },
             div {
                 id: "app-wrap",
+                onmouseenter: move |_| {
```

Comment:
- why resize the window on mouse enter?
- may want to check that the config has changed before writing it out to a file
- use `log` instead of `println`
- `minimal_view` criteria on line 445 is different from the criteria on line 392